regolith/helpers/l_grantshelper.py

```diff
@@ -16,32 +14,23 @@
     get_pi_id,
 )
 
-TARGET_COLL = "projecta"
-HELPER_TARGET = "l_projecta"
-ALLOWED_STATI = ["proposed", "started", "finished", "back_burner", "paused", "cancelled"]
-
-
+TARGET_COLL = "grants"
+HELPER_TARGET = "l_grants"
 
 def subparser(subpi):
-    subpi.add_argument("-v", "--verbose", action="store_true", help='increase verbosity of output')
-    subpi.add_argument("-l", "--lead",
-                       help="Filter milestones for this project lead"
-                       )
-    subpi.add_argument("-p", "--person",
-                       help="Filter milestones for this person whether lead or not"
+    subpi.add_argument("-v", "--verbose", action="store_true", help='outputs the grants with more information')
+    subpi.add_argument("-l", "--list",
```

Comment:
presumably the lister lists, so I doubt this arg is needed.  What is its purpose?